src/platform/linux/x11grab.cpp

```diff
@@ -797,7 +798,8 @@ namespace platf {
 
     BOOST_LOG(info) << "Detecting monitors"sv;
 
-    x11::xdisplay_t xdisplay { x11::OpenDisplay(nullptr) };
+    std::string display = config::video.output_name;  // get display name from config
+    x11::xdisplay_t xdisplay { x11::OpenDisplay(display.empty() ? nullptr : display.c_str()) };  // if config empty, nullptr will use the DISPLAY env var
```

Comment:
Is there a reason why we are only trying to get the display from the env variable, and not utilizing the config value?

I can revert this, and make it a separate PR if desirable.

Reply:
Moved to #2294